docs/advanced-topics/secure-metrics.asciidoc

```diff
@@ -0,0 +1,179 @@
+:page_id: secure-metrics
+ifdef::env-github[]
+****
+link:https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-{page_id}.html[View this document on the Elastic website]
+****
+endif::[]
+
+[id="{p}-{page_id}"]
+= Securing the ECK Operator metrics endpoint
+
+== Using the ECK Operator Helm Chart
+
+If you install ECK through the Helm chart commands listed in <<{p}-install-helm>>, you can now set `enableSecureMetrics` and `metricsBindAddress` and both RBAC and TLS/HTTPs will be enabled for the metrics endpoint.
+
+== Using the ECK Operator manifests
+
+If you install ECK through using the manifests using the commands listed in <<{p}-deploy-eck>> some additional changes will be required to enable secure metrics.
+
+* Enable the metrics endpoint in the `ConfigMap`.
+
+[source,sh]
+----
+kubectl get cm elastic-operator -n elastic-system -o yaml | sed "s|metrics-port: 0|metrics-port: 8081|" | kubectl apply -f -
```

Comment:
I do not like this particularly. Should we simply have the command to create an updated full `ConfigMap`?

Reply:
To me having a comment to create an updated full ConfigMap sounds better

Reply:
I've actually added a full configmap apply here instead, which just seems cleaner.